javascript/javascript3/week3/shopping-cart.js

```diff
@@ -0,0 +1,86 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    this.crrConvereted;
+    this.url = "https://api.exchangeratesapi.io/latest?base" + currency;
+    fetch(this.url)
+      .then((response) => response.json())
+      .then((ccr) => {
+        console.log(ccr);
+        this.crrConvereted = this.price / ccr.rates.DKK;
+        return console.log(this.crrConvereted);
+      });
+  }
+}
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+  addProduct(product) {
+    this.products.push(product);
+  }
+  removeProduct(product) {
+    this.products = this.products.filter((p) => p != product);
+  }
+  searchProduct(productName) {
+    return this.products.filter(
+      (product) => product.name.toLowerCase() === productName.toLowerCase()
+    );
+  }
+  getTotal() {
+    this.total = 0;
```

Comment:
It seems the variable 'total' scope is local to getTotal method, you don't need to add 'this' keyword in 'total' variable. if you add 'this' in front of a variable then it gets defined to the Class scope.